src/Builder/FrontendDialectHelper.cpp

```diff
@@ -103,6 +103,7 @@ static std::vector<T> CreateArrayAttribute(onnx::TensorProto initializer) {
   return std::vector<T>(&data[0], &data[0] + size);
 }
 
+#if 0
```

Comment:
Why not delete this block, rather than ifdef'ing it out?

Reply:
Forgot. Deleted now.